app/components/CrashDiagramCard.tsx

```diff
@@ -2,63 +2,165 @@ import { useState } from "react";
 import Alert from "react-bootstrap/Alert";
 import Card from "react-bootstrap/Card";
 import Image from "react-bootstrap/Image";
+import Button from "react-bootstrap/Button";
+import ButtonGroup from "react-bootstrap/ButtonGroup";
+import { BsZoomIn, BsZoomOut } from "react-icons/bs";
+import { SlActionUndo } from "react-icons/sl";
+import {
+  TransformWrapper,
+  TransformComponent,
+  useControls,
+} from "react-zoom-pan-pinch";
 import { Crash } from "@/types/crashes";
 
-// import axios from "axios";
```

Comment:
I zapped this axios library comment; I don't think we need it. Right?

Reply:
💯 🙏